code/src/main/java/com/onlinecontacttracing/servlets/delete-data/DeleteNotificationBatchServlet.java

```diff
@@ -0,0 +1,19 @@
+package com.onlinecontacttracing.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+@WebServlet("/delete-old-notification-batch")
+public class DeleteNotificationBatchServlet extends HttpServlet {
+
+  static final Logger log = Logger.getLogger(DeleteNotificationBatchServlet.class.getName());
```

Comment:
If you moved the logger into each concrete instance of DataDeleter, that'd allow you to not have to pass it in here.

Reply:
True, but every servlet needs to instantiate a logger based on its own class.